kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/prometheus/PrometheusIntegrationTest.java

```diff
@@ -136,4 +157,76 @@ public void testPrometheusProcessAndTaskMetrics() {
                 format("kie_server_task_exited_total{deployment_id=\"prometheus\",process_id=\"per-process-instance-project.usertask\",task_name=\"First task\",} %d.0", exitedTasks)
         );
     }
+
+    @Test
+    public void testPrometheusCaseMetrics() {
+        String caseId = startUserTaskCase(USER_YODA, USER_JOHN);
+        assertThat(caseId).isNotEmpty();
+
+        assertThat(getMetrics()).contains(
+              "kie_server_case_started_total{case_definition_id=\"" + CASE_DEF_ID + "\",",
+              "kie_server_case_running_total{case_definition_id=\"" + CASE_DEF_ID + "\",} 1.0"
+        );
+
+        caseClient.cancelCaseInstance(CONTAINER_ID_CASE, caseId);
+
+        assertThat(getMetrics()).contains(
+              "kie_server_case_started_total{case_definition_id=\"" + CASE_DEF_ID + "\",",
+              "kie_server_case_running_total{case_definition_id=\"" + CASE_DEF_ID + "\",} 0.0",
```

Comment:
@sutaakar Any potential timing issues here to get the counter to be 0?

Reply:
@vblagoje I am not sure if I get your question. The case is cancelled above so the number of running cases should decrement to 0.0. The decrement should happen in listener while cancelCaseInstance() is processed. There should be no timing issues.

Reply:
Ok cool. Just checking

Reply:
In the end  had to delete the check for counter value. Problem is that other tests don't close/delete cases correctly when they finish, causing counter to become inconsistent with actual opened cases.